app/components/util/HowDoYouFeel.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+const responseOptions = ['Awesome!', 'Just Okay', 'Unsatisfied']
+
+export default function HowDoYouFeel(props) {
+    //Isolate props and set initial state
+    const { disabled, className, onDone, ...otherProps } = props
+    const styleClasses = rankingStyleClasses(props)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.Howdoyoufeel, className)
+    const textClasses = textStyles()
+    const textClassName = cx(textClasses.textStyle)
+    const onSelection = e => {
+        onDone({ valid: true, value: e.target.value })
+    }
+    return (
+        <>
+            <p className={textClassName}>How do you feel about it</p>
+            <div className={cx(className, styleClasses.group, disabled)}{...otherProps}>
+                {responseOptions.map(option => {
+                    return (
+                        <label>
+                            <button
+                                onClick={onSelection}
+                                value={option}
+                                className={combinedClassName}
+                            >{option}</button>
+                        </label>
+                    )
+                })}
+            </div >
+        </>
+    )
+}
+
+const rankingStyleClasses = createUseStyles({
```

Comment:
createUseStyles should be called once, all the different properties can be in 1 object.  This will remove several lines of code above.